store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9769,4 +9769,13 @@ TODO: delete them permanently from here
   <desc>Whether to allow a user to access Zimbra X desktop</desc>
 </attr>
 
+<attr id="3081" name="zimbraFeatureModernMobileUIEnabled" type="boolean" cardinality="single" optionalIn="account,cos" flags="accountInherited" since="8.9.0">
```

Comment:
check if the id is already used in Zimbra X,  deprecate the earlier zimbraFeatureZXDesktopEnabled

Reply:
added deprecate for attribute zimbraFeatureZXDesktopEnabled

Reply:
3081 is in use by the common password config. This may be backported eventually - the collision might be annoying.

https://github.com/ZimbraOS/zm-mailbox/pull/212/files#diff-3b031f41b9ac7ae803e3e7cf389d1733R9981